.gitignore

```diff
@@ -5,3 +5,6 @@
 
 # helps Lennart configure his vim 
 pyrightconfig.json
+
```

Comment:
Can't comment directly on it so I'll write here - The `.DS_Store` stuff is still in the commit history here so can we delete them please? Also, note to remember to squash merge so we purge extra redundant file like this out of the commit history (since deleting it alone in subsequent commits won't be enough)

Reply:
we promise to squash merge so as to avoid using the bfg.

Reply:
You still need to make a commit to delete the file first, otherwise squash merge alone won't work as it won't have the delete commit to combine with to get rid of the file in the final squashed (single) commit

Reply:
Right, my bad. I still had the file in the root folder.